sdks/java/harness/src/main/java/org/apache/beam/fn/harness/FnHarness.java

```diff
@@ -262,6 +265,8 @@ public static void main(
                     }
                   });
 
+      MetricsEnvironment.setProcessWideContainer(new MetricsContainerImpl(null));
```

Comment:
Moving this to https://github.com/apache/beam/pull/14805

Unfortunately I could not make MetricsEnvironment instantiate a MetricsContainerImpl inside itself as this would create a dep cycle between their packages. So keeping this one as is.